scripts/dev.sh

```
@@ -0,0 +1,71 @@
+          --auto-stop 45 \
+          -e GH_PAT="$GH_PAT" \
+          | grep -oE '[0-9a-f\-]{36}')
+  fi
+fi
+
+# ──────────────────────── 4. verify GH_PAT on resumed boxes ─
+# (skip if we just created)
+if [[ -z "${NEWLY_CREATED:-}" && -n "${ID:-}" ]]; then
+  HAS_PAT=$(daytona sandbox list --format json 2>/dev/null |
+    jq -r --arg id "$ID" '.[] | select(.id==$id) | .env.GH_PAT // ""')
+  if [[ -z "$HAS_PAT" ]]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)